_schema.graphql

```diff
@@ -7729,6 +7732,9 @@ type Show implements Node {
   nearbyShows(
     sort: PartnerShowSorts
     status: EventStatus
+
+    # Whether to include local discovery stubs as well as displayable shows
+    discoverable: Boolean
```

Comment:
I'm glad we have this comment, because otherwise this wouldn't make sense - did you consider something like `includeStubs`?

Reply:
Yeah good point. It matches the name of the query param that this maps to over in Gravity.  "Discoverable" is now [one more scope](https://github.com/artsy/gravity/blob/master/app/api/v1/shows_endpoint.rb#L137) you can request, and it includes "displayable" as well as stub shows.

So it's consistent in a _sense_, but I do not dispute that the naming could be better 😬.